ansible/roles/test/tasks/tacacs_configure.yml

```diff
@@ -0,0 +1,23 @@
+# Set variables for setup
+- name: Set variables for TACACS+ setup
+  set_fact:
+    tacacs_server: "{{ tacacs_servers[0] }}"
+    tacacs_passkey: "test123"
+    tacacs_username: "test"
+    tacacs_passwd: "test"
+
+- name: Start TACACS+ service
+  become: true
+  service:
+    name: tac_plus
+    state: started
+  delegate_to: "{{ tacacs_server }}"
```

Comment:
suggest to move tacacs_server, passkey, test username, passwd all into test inventory. Make this as a testbed requirement, and leave the tacacs+ server setup outside of the test. 

I suggest also remove the starting tacacs+ service and add comment indicating whoever perform the testbed needs to setup tacacs service manually to enable the test user and passwd and passkey.

Reply:
Some comments have been added to inform user which setup need for TACACS+ testbed. The tacacs+ passkey, user account and password already moved to group_vars/lab/lab.yml